doc/api/errors.md

```diff
@@ -1310,8 +1310,14 @@ An invalid `options.protocol` was passed.
 <a id="ERR_INVALID_REPL_EVAL_CONFIG"></a>
 ### ERR_INVALID_REPL_EVAL_CONFIG
 
-Both `breakEvalOnSigint` and `eval` options were set in the REPL config, which
-is not supported.
+Both `breakEvalOnSigint` and `eval` options were set in the [`REPL`][] config,
+which is not supported.
+
+<a id="ERR_INVALID_REPL_INPUT"></a>
+### ERR_INVALID_REPL_INPUT
+
+The input may not be used in the [`REPL`][]. All prohibited inputs are
+documented in the [`REPL`][]'s documentation.
```

Comment:
Nit:  Instead of 

> All prohibited inputs are documented in the [`REPL`][]'s documentation.

maybe this?:

> The [`REPL`][] documentation specifies all prohibited inputs.

(Also, does it really?)

Reply:
Or maybe this?:

> See the [`REPL`][] documentation for more information.

Reply:
> does it really?

Two things are prohibited in the REPL: `uncaughtException` listeners in case the REPL is not running as standalone program and `process.setUncaughtExceptionCaptureCallback()`. Both of these cases are documented. I think it's not possible to use some module features but that's just a technical limitation so far.